pkg/controller/workflow/executor.go

```diff
@@ -5,6 +5,10 @@ import (
 	"fmt"
 	"time"
 
+	events2 "github.com/flyteorg/flytepropeller/pkg/controller/events"
```

Comment:
```suggestion
	controllerEvents "github.com/flyteorg/flytepropeller/pkg/controller/events"
```

Reply:
done, although if we're going to be strict about this there's a lot of existing foo2 that needs to be cleaned up

Reply:
Apologies and thank you 🙇🏽